### ISSUE TITLE
Add Apache-2 license

### ISSUE THREAD

**Issue by skmgoldin** [skmgoldin/tcr#11]
Does anyone object to Apache-2?

**Comment by kangarang** [comment]
I'm reading the Apache-2 license, but the words don't make sense to my small mind. Could anyone ELI5? @jamesyoung @GNSPS @skmgoldin @jlwaugh

**Comment by jlwaugh** [comment]
It's a _permissive_ free software license, which allows you to release modified parts of the code under any license. However, you are required to release all unmodified parts of the software under the Apache-2 license.

Additionally, every licensed file must contain any original copyright, patent, trademark and attribution notices in its redistributed code. Each modified file must also contain a notice about all the changes made to the original file.

**Comment by kangarang** [comment]
thank you for the great explanation @jlwaugh 

let's Apache-2 this badboy! 🥇
